prow/test/data/kubernetes-explicit-decoration-prow-job.yaml

```diff
@@ -41,5 +47,5 @@ spec:
   rerun_command: /test kubernetes-explicit-decoration
   type: presubmit
 status:
-  startTime: 2018-05-04T21:26:38Z
+  startTime: 2018-07-06T22:59:02Z
```

Comment:
What are these two files and why change the `startTIme`?

Reply:
These are used for the integration tests that you asked for :)

They are just autogenerated, the time is ignored when they are diffed but is recorded nonetheless.